web/gds-admin-ui/src/pages/app/lists/index.js

```diff
@@ -108,6 +109,26 @@ const VaspsList = (): React$Element<React$FragmentType> => {
         dispatch(fetchVasps())
     }, [dispatch])
 
+
+    function exportToCsv(rows) {
+        const { verified_contacts, ...rest } = rows[0]
+
+        let rowHeader = Object.keys(rest)
+
+        const _rows = rows.map(row => {
+            const { verified_contacts, ...rest } = row
+            return Object.values(rest)
+        })
+        _rows.unshift(rowHeader)
+
+        let csvFile = '';
+        for (let i = 0; i < _rows.length; i++) {
+            csvFile += generateCSV(_rows[i]);
+        }
+        const filename = `${dayjs().format("DD-MM-YY")}-vasps.csv`
```

Comment:
Could we change the filename to: `${dayjs().format("YYYY-MM-DD")}-directory.csv` please?